phones/apps.py

```diff
@@ -44,3 +48,19 @@ def twilio_validator(self) -> RequestValidator:
         if not settings.TWILIO_AUTH_TOKEN:
             raise Exception("Must define TWILIO_AUTH_TOKEN")
         return RequestValidator(settings.TWILIO_AUTH_TOKEN)
+
+    def send_iq_sms(self, to_num: str, from_num: str, text: str) -> None:
```

Comment:
Why is this a method on the AppConfig class? It seems like it would belong in `iq_utils.py` or similar.

Reply:
I'll move it to an `iq_utils.py` module. I just hate when I find myself making a `utils` module - I feel like I've probably failed in my architectural design or thinking somehow. But it makes sense.